sharktank/tests/models/llama/benchmark_amdgpu_test.py

```diff
@@ -173,6 +180,41 @@ def testBenchmark8B_f16_Decomposed(self):
             cwd=self.repo_root,
         )
 
+    @skipif_run_8b_llama
+    def testBenchmark8B_f16_Non_Decomposed_Prefill(self):
```

Comment:
This skip seems inverted to me. Skip 8b_f16 if "run_8b_llama" is set?

Do we have a central list of pytest marks? Looks like https://github.com/nod-ai/SHARK-Platform/blob/main/sharktank/conftest.py

I'd like to see a developer guide for how to run each set of tests and filter based on model family, model size, hardware required, etc. Once such a list of common usage exists it should be easier to square the workflow files against that (and shard across multiple runners, decide what to run on every commit vs nightly, etc.).